modules/carousel/jetpack-carousel.php

```diff
@@ -220,7 +220,7 @@ function enqueue_assets() {
 				'post_comment'         => __( 'Post Comment', 'jetpack' ),
 				'write_comment'        => __( 'Write a Comment...', 'jetpack' ),
 				'loading_comments'     => __( 'Loading Comments...', 'jetpack' ),
-				'download_original'    => sprintf( __( 'View full size <span class="photo-size">%1$s<span class="photo-size-times">&times;</span>%2$s</span>', 'jetpack' ), '{0}', '{1}' ),
+				'download_original'    => sprintf( wp_kses ( __( 'View full size <span class="photo-size">%1$s<span class="photo-size-times">&times;</span>%2$s</span>', 'jetpack' )), '{0}', '{1}' ),
```

Comment:
1. There's a space between `wp_kses` and `(` that shouldn't be there
2. `wp_kses` lacks the second argument so running this will throw
`Warning: Missing argument 2 for wp_kses()`
Please check `wp_kses` documentation https://codex.wordpress.org/Function_Reference/wp_kses